docs/dev/release_checklist.md

```diff
@@ -88,17 +87,26 @@ Please see the [official Poetry documentation on `version`](https://python-poetr
 !!! important
     The changelog must adhere to the [Keep a Changelog](https://keepachangelog.com/) style guide.
 
-This example uses 1.4.2, but change the version number to match the version you bumped to in the previous step. First, create a release branch off of `develop` (`git switch -c release-1.4.2 develop`).
+This guide uses `1.4.2` as the new version in its examples, so change it to match the version you bumped to in the previous step! Every. single. time. you. copy/paste commands :)
+
+First, create a release branch off of `develop` (`git switch -c release-1.4.2 develop`).
```

Comment:
Let's include an example with explicitly calling `git switch develop; git pull` as a prerequisite.

Reply:
There's a callout at the very start, but I added commands there as well for full clarity.